ui/app/AppLayouts/Profile/popups/AddSocialLinkModal.qml

```diff
@@ -0,0 +1,126 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import utils 1.0
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Popups 0.1
+
+import "../controls"
+
+StatusStackModal {
+    id: root
+
+    signal addLinkRequested(string linkText, string linkUrl, int linkType, string linkIcon)
+
+    implicitWidth: 480 // design
+    implicitHeight: 512 // design
+    anchors.centerIn: parent
+    padding: currentIndex === 0 ? 0 : Style.current.padding
+
+    header.title: qsTr("Add more links")
+    rightButtons: [finishButton]
+    finishButton: StatusButton {
+        text: qsTr("Add")
+        enabled: !!linkTarget.text
+        onClicked: {
+            root.addLinkRequested(d.selectedLinkTypeText || customTitle.text, // text for custom link, otherwise the link typeId
+                                  Utils.addSocialLinkPrefix(linkTarget.text, d.selectedLinkType),
```

Comment:
Does it make sense to have it in `ProfileUtils` instead?

Reply:
Yup, I meant to do it at one point but forgot, makes sense :)

Reply:
Added